partcad/src/partcad/project.py

```
@@ -241,19 +241,19 @@ def get_child_project_names(self):
             ):
                 children.append(self.name + "/" + subdir)
 
-        if "import" in self.config_obj and not self.config_obj["import"] is None:
-            imports = self.config_obj["import"]
+        if "dependencies" in self.config_obj and not self.config_obj["dependencies"] is None:
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)